File: app/src/main/java/net/xzos/UpgradeAll/json/cache/JSCacheData.java
package net.xzos.UpgradeAll.json.cache;

import org.json.JSONObject;

public class JSCacheData {

    private JSONObject jsoup_dom_dict = new JSONObject();
    private JSONObject http_response_dict = new JSONObject();

    public JSONObject getJsoupDomDict() {
        return jsoup_dom_dict;
    }

    public JSONObject getHttpResponseDict() {
        return http_response_dict;
    }
}

File: app/src/main/kotlin/net/xzos/upgradeAll/json/cache/JSCacheData.kt
package net.xzos.upgradeAll.json.cache

import org.jsoup.nodes.Document

data class JSCacheData(
        val httpResponseDict: MutableMap<String, String> = mutableMapOf(),
        val jsoupDomDict: MutableMap<String, Document> = mutableMapOf()
)
